src/OpenTelemetry.Api/Context/Propagation/B3Propagator.cs

```diff
@@ -25,7 +25,9 @@ namespace OpenTelemetry.Context.Propagation
 {
     /// <summary>
     /// A text map propagator for B3. See https://github.com/openzipkin/b3-propagation.
+    /// This class has been deprecated in favour of OpenTelemetry.Extensions.Propagators package.
     /// </summary>
+    [Obsolete("Use B3Propagator class from OpenTelemetry.Extensions.Propagators namespace, shipped as part of OpenTelemetry.Extensions.Propagators package.")]
```

Comment:
This message looks good to me. @alanwest to offer any additional feedback.

We need to use the same message in all Obsolete places, and also in changelog.

Reply:
I have updated changelog and added the same message to all obsolete places.